hack/deploy/agent/deployment.yaml

```diff
@@ -26,6 +26,9 @@ spec:
           - "--cluster-name=cluster1"
           - "--bootstrap-kubeconfig=/spoke/bootstrap/kubeconfig"
           - "--feature-gates=AddonManagement=true"
+        env:
+        - name: OPERATOR_NAME
+          value: multicluster-controlplane-agent
```

Comment:
use 

```
valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
```
?

Reply:
this is operator name. it is used by configurationpolicy to do some cleanups. they need deployment name instead of pod name.